### PR TITLE
fundtransfer in page object model

### PR DESCRIPTION
**Pull request by ravali-deriv**
This PR aims to add fundtransfer functionality of MT5 - Deposit and Withdrawal - USD to USD
the whole code is now in Page Object model framework

Clickup Card - https://app.clickup.com/t/20696747/CFDS-2826

### CODE REVIEW

**Comment by ravali-deriv** [comment]
This is the test run screenshot of the code

**Review by markodonnell-deriv — APPROVED** [review]
Looks good. 
The only issue is, we're trying to avoid using cy.Wait() but as I said to 
@jyothilakshmi-deriv 
 that it's best to just get a test in for now and then I can help see if I can work around the wait issue.

File: cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+const locators=require('../PageElements/mt5TransferLocators.json')
+
+class FundTransferPageObjects{
+    transferAmount="10";
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js:4]: we can remove the `;` in cypress code

**Review by meeyun-deriv — DISMISSED** [review]

File: cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js (hunk 1 above)
Comment (ravali-deriv) [reply cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js:4]: I have removed this

File: cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        cy.get(locators.fundTransferLocators.mt5AccountHandle)
+        .contains(this.mt5Account.split(" ")[0])
+        .next()
+        .contains(this.mt5Account.split(" ")[1])
+        .parentsUntil(locators.fundTransferLocators.mt5TradingDetails).next().children().invoke('text').as('balancebefore')
+
+        cy.get(locators.fundTransferLocators.usdBalanceLabel).eq(1).children().eq(0).invoke('text').as('usdbalancebefore')
+
+        cy.get('@usdbalancebefore').then(usdbalancebefore => {
+            usdbalancebefore = Number(usdbalancebefore.replace(/[^0-9.-]+/g,""))
```
[the remark this answers is not in the document]
Comment (ravali-deriv) [reply cypress/e2e/wip/CFD/Pages/fundTransferPageObjects.js:19]: I have changed all the variable names to camelCase

File: cypress/e2e/wip/CFD/PageElements/mt5TransferLocators.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        "mt5AccountHandle":"[class='trading-app-card__container trading-app-card--divider']",
+        "mt5TradingDetails":".trading-app-card__details",
+        "usdBalanceLabel":"[data-testid='dt_balance_text_container']",
+        "mt5TransferFromDropdown":"[class='dc-dropdown-container account-transfer-form__drop-down']",
+        "dropdownList":"[class='dc-dropdown__list dc-dropdown__list--enter-done']",
+        "dropdownEntry":"[data-testid='dti_list_item']",
+        "mt5TransferToDropdown":"[class='dc-dropdown-container account-transfer-form__drop-down account-transfer-form__drop-down--to-dropdown']",
+        "transferInputTestId":"dt_account_transfer_form_input",
+        "transferSubmitButton":"[data-testid='dt_account_transfer_form_submit']",
+        "closeButton":"[class='account-transfer-receipt__crypto--form-submit']",
```
[the remark this answers is not in the document]
Comment (ravali-deriv) [reply cypress/e2e/wip/CFD/PageElements/mt5TransferLocators.json:18]: @meeyun-deriv I have created this card for FE to improve the locators - https://app.clickup.com/t/20696747/TRAH-2645

**Pull request merged**